store/core/pom.xml

```diff
@@ -40,10 +46,41 @@
       <artifactId>scalatest_${scala.binary.version}</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
```

Comment:
It is better to put REST interface into a separate module, so that store-core is provide Store API only